vertica/datadog_checks/vertica/data/conf.yaml.example

```diff
@@ -0,0 +1,164 @@
+    ##
+    ## 1. true - `global_custom_queries` will override `custom_queries`
+    ## 2. false - `custom_queries` will override `global_custom_queries`
+    ## 2. extend - `global_custom_queries` will be used in addition to any `custom_queries`
+    #
+    # use_global_custom_queries: true
+
+    ## @param custom_queries - list - optional
+    ## Each query must have 2 fields:
+    ##
+    ## 1. query - This is the SQL to execute. It can be a simple statement or a multi-line script.
+    ## 2. columns - This is a list representing each column, ordered sequentially from left to right.
+    ##              The number of columns must equal the number of columns returned in the query.
+    ##              There are 2 required pieces of data:
+    ##                a. name - This is the suffix to append to `vertica.` in order to form
+    ##                          the full metric name. If `type` is `tag`, this column will
+    ##                          instead be considered a tag and will be applied to every
+    ##                          metric collected by this particular query.
+    ##                b. type - This is the submission method (gauge, monotonic_count, etc.).
+    ##                          This can also be set to `tag` to tag each metric in the row
+    ##                          with the name and value of the item in this column. You can
+    ##                          use the `count` type to perform aggregation for queries that
+    ##                          return multiple rows with the same or no tags.
+    ## 3. tags (optional) - A list of tags to apply to each metric.
+    #
+    # custom_queries:
+    #   - query: |  # Use the pipe if you require a multi-line script.
+    #       SELECT force_outer,
+    #              table_name
+    #       FROM v_catalog.tables
+    #     columns:
+    #       # Columns without a name are ignored, put this for any column you wish to skip:
+    #       # - {}
+    #       - name: table.force_outer
+    #         type: gauge
+    #       - name: table_name
+    #         type: tag
+    #     tags:
+    #       - test:vertica
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
+## service - mandatory - Name of the service that generated the log
+## source  - mandatory - Attribute that defines which Integration sent the logs
+## sourcecategory - optional - Multiple value attribute. Used to refine the source attribute
+## tags: - optional - Add tags to the collected logs
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - source: vertica
+#     type: file
+#     path: /catalog-path/database-name/node-name_catalog/vertica.log
```

Comment:
Are those Placeholders ? if so the path should be: `/<CATALOG_PATH>/<DATABASE_NAME>/<NODE_NAME_CATALOG>/vertica.log`